fe/fe-core/src/main/java/org/apache/doris/catalog/DatabaseIf.java

```diff
@@ -120,6 +120,16 @@ default T getTableOrMetaException(String tableName, TableIf.TableType tableType)
         return table;
     }
 
+    default T getTableViewOrMetaException(String tableName) throws MetaNotFoundException {
+        T table = getTableOrMetaException(tableName);
+        if (table.getType() != TableIf.TableType.OLAP && table.getType() != TableIf.TableType.MATERIALIZED_VIEW) {
+            throw new MetaNotFoundException(
+                    "table type is not olap or materialized view, tableName=" + tableName + ", type="
+                            + table.getType());
+        }
+        return table;
+    }
+
```

Comment:
I think the following interface is better due to it is more generic.
```suggestion
    default T getTableOrMetaException(String tableName, List<TableIf.TableType> tableTypes)
            throws MetaNotFoundException {
        T table = getTableOrMetaException(tableName);
        if (!tableTypes.contains(table.getType())) {
            throw new MetaNotFoundException(
                    "Tye type of " + tableName + " doesn't match, expected data tables=" + tableTypes);
        }
        return table;
    }
```